### DISCUSSION TITLE
How can I increase height of text in LineEdit Widget ?

### DISCUSSION THREAD

**Discussion by dridk** [ceccopierangiolieugenio/pyTermTk#269] [Q&A] [answered]
Text in LineEdit is very small compared with Textual. Can I increate the height is some way ?

**Comment by ceccopierangiolieugenio** [comment, marked as answer]
You can use the **textedit** instead of **lineedit**.
**lineedit** is somewhat a subset of the **textedit** features.

Or you can use the **TTkTextPicker** that internally use the textedit:

https://github.com/user-attachments/assets/d76d9fcd-9e3b-4792-b641-8ef5042aea21

I can provide you an example if you have something specific in mind.
